frontend/src/pages/associations/elections/ActiveList.tsx

```diff
@@ -0,0 +1,246 @@
+import {PageTitle} from "../../../utils/common";
+import React, {useContext, useState} from "react";
+import {Link} from "react-router-dom";
+import {Button, Col, Modal} from "react-bootstrap";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import {activeStatus, api, useBetterQuery} from "../../../services/apiService";
+import {LoadingAssociation} from "../Loading";
+import {Election, Ballot, Choice} from "../../../models/associations/election";
+import {ToastContext} from "../../../utils/Toast";
+import {ListOfVotersButtonModal} from "./Commons";
+
+export const AssociationElectionActiveList = ({ association }) => {
+
+    const associationId = association.id;
+    const { data, status, error } = useBetterQuery<Election[]>(
+        "elections.list",
+        api.elections.list,
+        associationId,
+        activeStatus.Current
+    );
+
+    if (status === "loading") return <LoadingAssociation />;
+    else if (status === "error") return `Something went wrong: ${error}`;
+    else if (data) {
+        return (
+            <>
+                <PageTitle className={"mt-6"}>Élections en cours</PageTitle>
+                <Row>
+                    {data.map(election => {
+                        return (
+                            <Col md={5} key={election.id}>
+                                <ElectionCard election={election} association={association}/>
+                            </Col>
+                        );
+                    })}
+                </Row>
+            </>
+        )
+    }
+};
+
+const ElectionCard = ({ election, association }) => {
+    let editButton;
+    if (association.myRole.electionPermission) {
+        editButton = (
```

Comment:
Why not using a `<Button>` ?
Also, I think it would look better without the text

Reply:
OK, that'll do for now